tests/introspection/iOS/iOSApiSelectorTest.cs

```diff
@@ -158,6 +158,14 @@ protected override bool CheckResponse (bool value, Type actualType, MethodBase m
 			var declaredType = method.DeclaringType;
 
 			switch (declaredType.Name) {
+			case "AVUrlAsset":
+				switch (name) {
```

Comment:
Small issue with the case indentation. Look at the others.

Reply:
nice catch, thanks!